packages/web3-eth-contract/src/encoders/EventFilterEncoder.js

```diff
@@ -22,6 +22,21 @@
 
 import isArray from 'lodash/isArray';
 
+/**
+ * @param {String} type
+ * @param {String | Number} param
+ * @param {AbiCoder} abiCoder
+ *
+ * @returns {String} encoded plain param
+ */
+const encodeParameter = (type, param, abiCoder) => {
```

Comment:
On the second look after my vacation is it probably better to extend the ``AbiCoder`` with an ``encodeTopic`` method and to handle all dynamically allocated types. Sorry.

Private methods do currently not in web3.js but will be added asap. 

Proposals for private properties and methods:
- https://github.com/tc39/proposal-private-methods 
    + https://babeljs.io/docs/en/babel-plugin-proposal-private-methods
- https://github.com/tc39/proposal-private-fields
    + https://babeljs.io/docs/en/babel-plugin-proposal-class-properties